stimela/cargo/cab/cubical/parameters.json

```diff
@@ -435,13 +435,13 @@
             ], 
             "default": null, 
             "name": "out-subtract-dirs"
-        },
+        }, 
         {
-            "name" : "out-casa-gaintables",
-            "info": "Export gaintables to CASA caltable format. Tables are exported to same directory as set for cubical databases",
-            "dtype": "bool",
-            "default": false
-        },
+            "info": "Export gaintables to CASA caltable format. Tables are exported to same directory as set for cubical databases", 
+            "dtype": "bool", 
+            "default": false, 
```

Comment:
Default is true according to parset.

Reply:
It's false here because it was broken. But no need to have the same information twice, so if the user wants to use the CASA tables they'll have to explicitly set enable it.

Reply:
That was a long time ago - cubical databases are extremely hard to interact with. If we want to inspect self cal solutions I would suggest enabling this.